pages/collection-list.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: collection-list
+template-title: Collection list
+template-description: List of collection with metadata, search bar, filter, user avatars, and tags. 
+title: Requirements
+description: In order to be certified as a PRP, you need to satisfy all of the requirements. For each requirement, submit proper evidence or responses. If you have questions, please reach out to your assigned Certifying Agent. 
+source-domain: PRP intake questionnaire
+source-url: http://cbprs.org/wp-content/uploads/2020/08/PRP-Intake-Questionnaire.pdf
+figma:
+collection:
+   - title: "1: Implement an information security policy"
+     items:
+       - Comment by Erika Spoon
+       - 3 hours ago
+     image-source: https://images.unsplash.com/photo-1597223557154-721c1cecc4b0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Not Started
+     status-color: bg-primary-lighter
+   - title: "2: Physical, technical and administrative safeguards"
+     items:
+       - Status changed by Ben Cunningham
+       - 4 days ago
+     image-source: https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Revision Needed
+     status-color: bg-secondary-light
```

Comment:
Overall, really nice use of yml for this content! One thing you might consider is using a conditional in the template for the tag color. That way content editor doesn't have to decide each time or risk inconsistency. You could do it as an include to keep the template clean. In pseudo code, "if this status, then this bg-color".